### PR TITLE
[Converter] incoming data format

### PR DESCRIPTION
**Pull request by jaeyun-jung**
Update tensor dimension of incoming data, 0-init.
 - video : [ch][w][h][frames]
 - audio : [ch][samples]
 - text : [size][samples]
 - octet/tensor : [size]

### CODE REVIEW

**Comment by jaeyun-jung** [comment]
@niley7464 please check this PR and update tct if this makes an error.

File: tests/nnstreamer_sink/unittest_sink.cc [hunk 1]
```diff
@@ -2690,8 +2689,6 @@ TEST (tensorStreamTest, audioS8)
   EXPECT_EQ (g_test_data.tensors_config.info.info[0].type, _NNS_INT8);
   EXPECT_EQ (g_test_data.tensors_config.info.info[0].dimension[0], 1U);
   EXPECT_EQ (g_test_data.tensors_config.info.info[0].dimension[1], 500U);
-  EXPECT_EQ (g_test_data.tensors_config.info.info[0].dimension[2], 1U);
```
Comment (myungjoo) [inline tests/nnstreamer_sink/unittest_sink.cc:2693]: How about
```EXPECT_LE (g_test_data.tensors_config.info.info[0].dimension[2], 1U);```
?

**Comment by niley7464** [comment]
> @niley7464 please check this PR and update tct if this makes an error.

TCT does not failed :)
- NativeTCT 20230823
- csharp-tct 20230828
- web-tct 20230825

File: tests/nnstreamer_sink/unittest_sink.cc (hunk 1 above)
Comment (jaeyun-jung) [reply tests/nnstreamer_sink/unittest_sink.cc:2693]: updated

**Review by anyj0527 — APPROVED** [review]
LGTM👍

**Review by niley7464 — APPROVED** [review]
LGTM!

**Review by songgot — APPROVED** [review]
LGTM

**Review by again4you — APPROVED** [review]
Verified!

**Pull request merged**